class03 aws content (#143)
devopsacademyau/academy: classes/02class/exercises/c02-network12-challenge/README.md

# AWS VPC + subnets

Commit all your answers to a text file `c02-network12.txt` under your folder name.

Everything is to be done preferable on Sydney region `ap-southeast-2`

## VPC Peering

> Once you have your custom VPC + subnets + nacls + security groups in place and tested (up to exercise c02-network-11 completed), pair with someone else and create a VPC peering within your vpc's.

Reviewer: ```suggestion
Once you have your custom VPC + Subnets + NACLs + security groups in place and tested (up to exercise c02-network-11 completed), pair with someone else and create a VPC peering between your VPCs.
```

> Few wording assumed from now:

Reviewer: ```suggestion
Few definitions assumed in this exercise:
```

> - your vpc = origin vpc

Reviewer: ```suggestion
- your VPC = origin VPC
```

> - someone else vpc = target vpc

Reviewer: ```suggestion
- someone else's VPC = target VPC
```

#### Deliverables

- explain how was the process to create a VPC peering
- add routes from origin VPC to target VPC - only for private subnets
- open any network ACL to allow the connection within the subnets -- try to only open the mininum required

> - from an instance in one vpc, SSH to the other instance (will have to share the ssh keys, create a new one just for this exercise -- pay attention to security)

Reviewer: ```suggestion
- from an instance in one VPC, SSH to the other instance (will have to share the SSH keys, create a new one just for this exercise -- pay attention to security)
```

> - open any security group to allow the SSH within the instances

Reviewer: ```suggestion
- open any security group to allow the SSH communication between instances
```
